overlord/overlord_test.go

```diff
@@ -643,7 +668,10 @@ func (ovs *overlordSuite) TestOverlordStartUpSetsStartOfOperation(c *C) {
 }
 
 func (ovs *overlordSuite) TestEnsureLoopPruneDoesntAbortShortlyAfterStartOfOperation(c *C) {
-	restoreIntv := overlord.MockPruneInterval(100*time.Millisecond, 1000*time.Millisecond, 1*time.Hour)
+	w, restoreTicker := fakePruneTicker()
+	defer restoreTicker()
+
+	restoreIntv := overlord.MockPruneInterval(1*time.Millisecond, 1000*time.Millisecond, 1*time.Hour)
```

Comment:
Does this matter, do we need this at all? I mean, we mock the pruneTicker in the lines above so the pruneInterval is irrelevant anyway - or am I reading the code wrong maybe?